src/Tribe/Ticket_Object.php

```diff
@@ -834,6 +917,20 @@ public function show_description() {
 			// Make sure we have the correct value
 			return tribe_is_truthy( $show );
 		}
+
+		/**
+		 * Access the ID of the Event parent of the current Ticket.
+		 *
+		 * @since TBD
+		 *
+		 * @return int|null
+		 */
+		public function get_event_id() {
+			if ( is_null( $this->event_id ) ) {
+				$this->event_id = $this->get_event();
```

Comment:
we should be doing `$this->get_event()->ID`  because it returns a `WP_Post` object.